src/routes/_fragments/Rsvp.svelte

```diff
@@ -28,7 +28,7 @@
 		</form>
 	</div>
 {:else}
-	<a class="buttony" href="/account">Log in to register</a>
+	<p>You need to log in to register for events</p>
```

Comment:
```suggestion
	<br />
	<a href="/account">You need to log in to register for events</a>
```

Reply:
![image](https://github.com/mallsoft/eventus/assets/36839009/350a4891-1dee-4919-a019-4fbfbe099be7)

Reply:
Looks the same, but is clickable